libs/SalesforceSDKCore/SalesforceSDKCore/Classes/Login/SFLoginViewController.m

```diff
@@ -317,8 +317,8 @@ - (UIBarButtonItem *)createSettingsButton {
                                                  handler:^(__kindof UIAction* _Nonnull action) {
             UIViewController *configPicker = [BootConfigPickerViewController makeViewControllerOnConfigurationCompleted:^{
                 [self dismissViewControllerAnimated:YES completion:^{
-                    if ([self.delegate respondsToSelector:@selector(loginViewControllerDidReload:)]) {
-                        [self.delegate loginViewControllerDidReload:self];
+                    if ([self.delegate respondsToSelector:@selector(loginViewControllerDidChangeLoginOptions:)]) {
```

Comment:
Reload is not enough - we need the auth request within the auth session to be recreated (see earlier comment).